test/test_main.py

```diff
@@ -26,8 +33,11 @@ def testRootXHeader(self):
         self.assertIn('rebuildingtogether.staff@gmail.com', str(response))
 
 
-class StatefulTest(unittest2.TestCase):
+class StatefulTest(unittest.TestCase):
     def setUp(self):
+        app_engine_test_utils.activate_app_engine_testbed()
+        app_engine_test_utils.clear_ndb_cache()
+        self.keys = test_models.CreateAll()
         test_models.CreateAll()        
```

Comment:
this line looks redundant with the one above.

Reply:
Oops! Thanks